code/modules/jobs/job_types/bos.dm

```diff
@@ -519,11 +520,11 @@ Senior Knight
 /datum/outfit/loadout/skb
 	name = "Senior Knight-Marksman"
 	backpack_contents = list(
-		/obj/item/gun/ballistic/automatic/marksman = 1,
-		/obj/item/ammo_box/magazine/m556/rifle = 3,
-		/obj/item/gun/ballistic/automatic/pistol/sig = 1,
-		/obj/item/ammo_box/magazine/m9mm = 2,
-		/obj/item/binoculars = 1
+		/obj/item/gun/ballistic/automatic/slr = 1,
```

Comment:
Any idea why we never see this gun in the game playa... cause it's NOT IN THE SPAWNERS CAUSE IT'S NOT CANON